magento/pwa-studio
# [PWA-2074] Change image placeholder to svg for Lighthouse improvement

## Description
Change transparent placeholder image from a png to an svg so we avoid penalty's in Lighthouse for serving an undersized image. Since SVGs are scalable, we can serve a small image without getting points deducted.

## Related Issue

Closes [PWA-2074](https://jira.corp.magento.com/browse/PWA-2074)

## Acceptance





### Verification Stakeholders



## Verification Steps






#### Test scenario(s) for direct fix/feature

- Navigate to category page
- Placeholder image should be visible for a brief moment before images load (use network throttle if need be)
- Run lighthouse in devtools
- Best Practices should no longer mention "Serves images with low resolution"

#### Test scenario(s) for any existing impacted features/areas

Validate all areas where placeholder is used (logo, category/image tiles, PDP page) to ensure sizing/functionality of lazy load still works as intended

## Breaking Changes (if any)
None

## Checklist



- [x] I have update tests to cover my changes, if necessary.
